packages/mdc-toolbar/foundation.ts

```diff
@@ -220,11 +255,13 @@ export default class MDCToolbarFoundation extends MDCFoundation {
     }
   }
 
-  updateElementStylesDefaultBehavior_(flexibleExpansionRatio) {
-    const maxTitleSize = MDCToolbarFoundation.numbers.MAX_TITLE_SIZE;
-    const minTitleSize = MDCToolbarFoundation.numbers.MIN_TITLE_SIZE;
+  private updateElementStylesDefaultBehavior_(flexibleExpansionRatio: number) {
+    const maxTitleSize = numbers.MAX_TITLE_SIZE;
+    const minTitleSize = numbers.MIN_TITLE_SIZE;
     const currentTitleSize = (maxTitleSize - minTitleSize) * flexibleExpansionRatio + minTitleSize;
 
     this.adapter_.setStyleForTitleElement('font-size', `${currentTitleSize}rem`);
   }
 }
+
+export default MDCToolbarFoundation;
```

Comment:
change to export {...as default, ...}

Reply:
I actually switched to this split export syntax to make it easier to disable the default export in the future. See #4407 PR description for an explanation.